src/bionics.cpp

```diff
@@ -85,20 +203,26 @@ bionic_data const& bionic_info(std::string const &id)
 
     debugmsg("bad bionic id");
 
-    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0, "bad_bionic", false};
+    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
+                                         "bad_bionic", false,
+                                         { num_bp , 1 } };
```

Comment:
Double the curly brackets `{{ num_bp, 1 }}` to specify the expected map of pairs instead of a single pair. BTW why do you declare the map as `<body_part, int>`. You already have your `cbm_pair` `<body_part, size_t>`. Are they for different purposes?